src/js/editor/editor.js

```diff
@@ -571,8 +520,9 @@ class Editor {
     section.renderNode.markDirty();
   }
 
-  reparseSection(section) {
-    this._parser.reparseSection(section, this._renderTree);
+  _reparseCurentSection() {
+    const {headSection:currentSection } = this.cursor.offsets;
```

Comment:
I'm OK with punting the removal of cursors, since there was already something in the parser re: cursors before. Let's just keep the goalpost in mind for now.